src/lib/packages.ts

```diff
@@ -136,6 +168,14 @@ export async function hoist(packageName: string): Promise<void> {
   await writeRootPackage(rootPkg);
 }
 
+export namespace hoist {
+  /**
+   * Options for the hoist function
```

Comment:
I'm not sure if this has been documented yet but we are moving to:

```
/** One liners */
/**
 * Multi
 * Line
 */

Reply:
Let's get that into prettier or tslint. Unless there's an easy way to get the editor to format these differently, I'll likely continue like this thanks to all the stuff vscode inserts automatically.